codegenerator/cli/src/executor/init.rs

```diff
@@ -236,11 +236,10 @@ pub async fn run_init_args(init_args: InitArgs, project_paths: &ProjectPaths) ->
         } else {
             // Else install the local version for development and testing
             match env::current_exe() {
-                // This should be something like "~/envio/hyperindex/codegenerator/target/debug/envio" or ".../target/debug/integration_tests"
-                Ok(exe_path) => exe_path
-                    .join("../../../cli/npm/envio")
-                    .to_string_lossy()
-                    .to_string(),
+                // This should be something like "file:~/envio/hyperindex/codegenerator/target/debug/envio" or "file:.../target/debug/integration_tests"
+                Ok(exe_path) => {
+                    format!("file:{}/../../../cli/npm/envio", exe_path.to_string_lossy())
```

Comment:
By default it's installed using the `link:` protocol instead of file, but it makes the dependency resolution a little bit weird. The `file:` protocol makes pnpm treat the local envio as a real package.

Btw, if we add it for the `./generated` we won't need to have a `pnpm-workspaces.yaml` file.

Reply:
Btw in path_utils there's a function called normalize_path which will get rid of unneeded ../../ etc in the middle of your path. Doesn't matter too much but it's there.

Reply:
Awesome, never new about the "file:" protocol.